tommath_class.h

```diff
@@ -624,9 +636,10 @@
 #endif
 
 #if defined(BN_MP_MUL_C)
+#   define BN_FAST_S_MP_MUL_DIGS_C
```

Comment:
lulz looks like that comment mentioning "fast_s_mp_mul_digs" triggers `dep.pl`

Reply:
Oh, great \*sigh\*
Had merge conflicts with `tommath_class.h`and `callgraph.txt` and thought the simplest way would be to just delete their content and let `make new_file` do its thing.
Which *seemed* to work.
What to do: change my comment or patch dep.pl to ignore comments (which is quite complicated if you want to get all edgecases)?

Reply:
>  Which _seemed_ to work.

it worked indeed.

> What to do: change my comment or patch dep.pl to ignore comments

very good question!
@karel-m you're the perl guy I know, is there an easy way to fix this in dep.pl?